airbyte-webapp/src/components/PageTitle/PageTitle.tsx

```diff
@@ -29,6 +29,15 @@ export const MiddleBlock = styled.div`
   justify-content: center;
 `;
 
+export const MiddleTitleBlock = styled(H3)`
```

Comment:
While I'm not implementing all of the design changes for the main content of the Connector pages, centering the titles made things feel more natural with the docs panel.